.github/workflows/build.yml

```
@@ -116,7 +116,7 @@ jobs:
         [ -z "$AWS_ACCESS_KEY_ID" ] || aws s3 cp mpy-cross/mpy-cross.static-raspbian s3://adafruit-circuit-python/bin/mpy-cross/mpy-cross.static-raspbian-${{ env.CP_VERSION }} --no-progress --region us-east-1
         [ -z "$AWS_ACCESS_KEY_ID" ] || aws s3 cp mpy-cross/mpy-cross.static s3://adafruit-circuit-python/bin/mpy-cross/mpy-cross.static-amd64-linux-${{ env.CP_VERSION }} --no-progress --region us-east-1
         [ -z "$AWS_ACCESS_KEY_ID" ] || aws s3 cp mpy-cross/mpy-cross.static.exe s3://adafruit-circuit-python/bin/mpy-cross/mpy-cross.static-x64-windows-${{ env.CP_VERSION }}.exe --no-progress --region us-east-1
-        zip -9 circuitpython-stubs.CP_VERSION }}.zip stubs
+        zip -9 circuitpython-stubs.${{ env.CP_VERSION }}.zip stubs
```

I think this should not include the version: the version is added when it's uploaded.
```suggestion
        zip -9 circuitpython-stubs.zip stubs
```

---

of course you're right